src/web/mjs/connectors/templates/MH.mjs

```diff
@@ -0,0 +1,69 @@
+import Connector from '../../engine/Connector.mjs';
+import Manga from '../../engine/Manga.mjs';
+
+export default class MH extends Connector {
+
+    constructor() {
+        super();
+        super.id = undefined;
+        super.label = undefined;
+        this.tags = [];
+        this.url = undefined;
+
+        this.path = '/booklist?page=';
+        this.pathMatch = 'page=(\\d+)';
```

Comment:
I'm surprised this works since this is a string.
However i would still recommend to use a regex literal.
`/page=(\d+)/`

Reply:
May also rename to `queryMangasPageCountMatcher`, to imply that this belongs to the logic that extract the manga page count from the pagination link